torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -370,6 +372,38 @@ void IrPrinter::handle(const Merge* m) {
   os_ << "\n";
 }
 
+void IrTransformPrinter::handle(Fusion* f) {
+  auto all_vals = DependencyCheck::getAllValsBetween(
+      {f->inputs().begin(), f->inputs().end()}, f->outputs());
```

Comment:
Should just use `fusion->usedMathVals()`

Reply:
Updated. Thanks!